packages/nartus_ui_package/lib/widgets/alert_dialog/nartus_alert_dialog.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:nartus_ui_package/dimens/dimens.dart';
+import 'package:nartus_ui_package/nartus_ui.dart';
+
+class NartusAlertDialog extends StatelessWidget {
+  final String? iconPath;
+  final String title;
+  final String content;
+  final String primaryButtonText;
+  final VoidCallback onPrimaryButtonSelected;
+  final String? secondaryButtonText;
+  final VoidCallback? onSecondButtonSelected;
+  final String? textButtonText;
+  final VoidCallback? onTextButtonSelected;
+
+  const NartusAlertDialog(
+      {required this.title,
+      required this.content,
+      required this.primaryButtonText,
+      required this.onPrimaryButtonSelected,
+      Key? key,
+      this.iconPath,
+      this.secondaryButtonText,
+      this.onSecondButtonSelected,
+      this.textButtonText,
+      this.onTextButtonSelected})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.fromLTRB(24, 40, 24, 40),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
```

Comment:
u can use `crossAxisAlignment: CrossAxisAlignment.stretch` here, and remove the other column in line 90, or the SizedBox in line 83